source/content.css

```diff
@@ -1079,3 +1079,9 @@ body > .footer li a {
 .dropdown-menu-sw {
 	transform-origin: 90% top;
 }
+
+/* Fix wrapping text on tabs when gists tab is enabled on profile #1757 */
+nav.UnderlineNav-body a:nth-last-child(n+7).UnderlineNav-item,
+nav.UnderlineNav-body a:nth-last-child(n+7).UnderlineNav-item ~ * {
```

Comment:
You probably don’t need `nav.UnderlineNav-body a` in both. Use `.UnderlineNav-item:nth-last-child(n+7)`

Reply:
@sindresorhus is there a stylelint rule to enforce the order in compound selectors? e.g. `.hello:nth-child(1)` instead of `:nth-child(1).hello`

Reply:
I went with something specific as I wasn't sure everywhere this class was used, I can see it both ways.

Reply:
> is there a stylelint rule to enforce the order in compound selectors? e.g. `.hello:nth-child(1)` instead of `:nth-child(1).hello`

No, but you should request it. That would be a useful rule.